code/game/objects/structures/tables_racks.dm

```diff
@@ -175,6 +175,26 @@
 			return
 		// If the tray IS empty, continue on (tray will be placed on the table like other items)
 
+	if(istype(I, /obj/item/riding_offhand))
+		var/obj/item/riding_offhand/riding_item = I
+		var/mob/living/carried_mob = riding_item.rider
+		if(carried_mob == user) //Piggyback user.
+			return
+		switch(user.a_intent)
+			if(INTENT_HARM)
+				user.unbuckle_mob(carried_mob)
+				tableheadsmash(user, carried_mob)
+			if(INTENT_HELP)
+				carried_mob.visible_message("<span class='notice'>[user] begins to place [carried_mob] onto [src]...</span>", \
+					"<span class='userdanger'>[user] begins to place [carried_mob] onto [src]...</span>")
+				if(do_after(user, 3.5 SECONDS, target = carried_mob))

```

Comment:
Maybe make it scalable by gloves like on pickup? If not.

Reply:
Done. Didn't outright remove the delay on expert because I don't want to have big impact changes, but that can be tweaked as necessary.